tests/Aspire.Dashboard.Tests/Model/ResourceSourceViewModelTests.cs

```diff
@@ -94,7 +96,7 @@ void AddStringProperty(string propertyName, string? propertyValue)
                 value: "project",
                 contentAfterValue: [new LaunchArgument("arg2", true), new LaunchArgument("--key", true), new LaunchArgument("secret", false)],
                 valueToVisualize: "path/to/project arg2 --key secret",
-                tooltip: "path/to/project arg2 --key secret"));
+                tooltip: $"path/to/project arg2 --key {WebUtility.HtmlDecode(DashboardUIHelpers.GetMaskingText(6).Value)}"));
```

Comment:
Are there other tests that should be added for this change?

Reply:
Added those tests in `ExpressionResolverTests`